src/components/TokenClaimDialog.tsx

```diff
@@ -319,6 +355,26 @@ function useRewardClaims() {
     },
     [chainId, account, rewardsContract, updateClaimStatus],
   )
+
+  const claimGaugeReward = useCallback(
+    async (gauge: Gauge) => {
+      if (!chainId || !account || !minterContract) return
+      try {
+        updateClaimStatus(gauge.poolName, STATUSES.PENDING)
+        const txn = await minterContract["mint(address)"](gauge.address)
```

Comment:
any reason to use `["mint(address)"]` here instead of just `.mint(gauge.address)`? It doesn't look like this function is overloaded in the abi.

Reply:
I was having issues with this call but it was related to having a gas property in the Minter ABI. I can change this back to `.mint`